modules/azure-auth-header/azure-auth-header.c

```diff
@@ -69,7 +68,7 @@ _azure_auth_header_get_str_to_hash(AzureAuthHeaderPlugin *self, glong content_le
 static gsize
 _azure_auth_header_get_digest(AzureAuthHeaderPlugin *self, guint8 **digest, const GString *input)
 {
-  gsize digest_len = g_checksum_type_get_length(self->alg);
+  gsize digest_len = g_checksum_type_get_length(G_CHECKSUM_SHA256);
```

Comment:
This patch can be squashed into the original patch. I do not see much reason to keep it separate.